components/style/restyle_hints.rs

```diff
@@ -330,27 +337,52 @@ impl Sensitivities {
 /// This allows us to quickly scan through the dependency sites of all style
 /// rules and determine the maximum effect that a given state or attribute
 /// change may have on the style of elements in the document.
+///
+/// Note that there are measurable perf wins from storing them separately, and
```

Comment:
I don't think the meaning of "storing them separately" is clear in this context.